core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -109,10 +161,16 @@ enum class Axis(val int: Int) {
 	}
 }
 
+/**
+ * The six orthogonal vectors corresponding to the faces of a cube.
```

Comment:
To clarify: these are the faces of the cube _as "seen" from the center_; a diagram would be helpful, but failing that, the `PlanarFace.XP` is the "face found by a ray exiting the center of a cube toward positive X."